src/Microsoft.Extensions.Logging.Abstractions/breakingchanges.netcore.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "TypeId": "public class Microsoft.Extensions.Logging.Abstractions.NullLogger<T0> : Microsoft.Extensions.Logging.ILogger<T0>",
```

Comment:
This type exists - https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging.Abstractions/NullLoggerOfT.cs. Why is it marked as removal?

Reply:
Is this because the baseline was incorrect to start with?

Reply:
We think it's a bug in the tool, which is logged, and we have another issue to revert these "fake" breaking changes.